source/dotnet/Library/AdaptiveCards/AdaptiveTypedBaseElementConverter.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AdaptiveCards
+{
+    public abstract class AdaptiveTypedBaseElementConverter : JsonConverter
+    {
+        protected ParseContext parseContext;
+        public ParseContext ParseContext
```

Comment:
Curious: Wouldnt automatic properties work here?Also why is the local member (parseContext) declared as protected? Additionally why is ParseContext - the property a public property vs protected?